mpyl-example.py

```diff
@@ -10,13 +10,13 @@ def main(log: Logger, args: argparse.Namespace):
         from src.mpyl.steps.models import RunProperties
         from src.mpyl.utilities.pyaml_env import parse_config
         from src.mpyl.cli import MpylCliParameters
-        from src.mpyl.build import run_mpyl
+        from src.mpyl.cli.commands.build.build import run_mpyl
```

Comment:
Hmm, why did we do that 🤔 . All the other command execution code is there too.

Reply:
Because these functions are not only used in the cli, plus we thought the 'main' mpyl function (`run_mpyl`) shouldn't be hidden so far into a subfolder.